packages/lesswrong/components/editor/EditLinkpostUrl.tsx

```diff
@@ -67,7 +67,7 @@ const EditLinkpostUrl = ({
       <Input
         inputRef={inputRef}
         className={classes.input}
-        value={(document && document[path]) || defaultValue || ""}
+        value={(document[path]) || defaultValue || ""}
```

Comment:
Is this change safe?

Reply:
I removed it because I was getting a "document is null-checked here, but not earlier" warning. But, I'm not actually that confident it should resolve in this direction.

Reply:
See line 50